File: BMI_Calculator.cs
using System;

class Program
{
    static void Main(string[] args)
    {
        Console.Write("Enter weight in Kilograms: ");
        double weight = Convert.ToDouble(Console.ReadLine());

        Console.Write("Enter height in Meters: ");
        double height = Convert.ToDouble(Console.ReadLine());

        double bmi = weight / Math.Pow(height, 2);

        Console.WriteLine("Your BMI is " + Math.Round(bmi, 2));

        if (bmi < 18.5)
        {
            Console.WriteLine("You are underweight.");
        }
        else if (bmi < 24.9)
        {
            Console.WriteLine("You are normal weight.");
        }
        else if (bmi < 29.9)
        {
            Console.WriteLine("You are overweight.");
        }
        else
        {
            Console.WriteLine("You are obese.");
        }
    }
}

File: BMI_Calculator.py
def calculate_bmi():
    height = float(input("Enter your height in meters: "))
    weight = float(input("Enter your weight in kg: "))
    bmi = weight / (height ** 2)
    print("Your BMI is: {0} and you are: ".format(bmi), end='')

    if bmi < 15:
        print("Very severely underweight")
    elif 15 <= bmi < 16:
        print("Severely underweight")
    elif 16 <= bmi < 18.5:
        print("Underweight")
    elif 18.5 <= bmi < 25:
        print("Normal (healthy weight)")
    elif 25 <= bmi < 30:
        print("Overweight")
    elif 30 <= bmi < 35:
        print("Moderately obese")
    elif 35 <= bmi < 40:
        print("Severely obese")
    else:
        print("Very severely obese")

calculate_bmi()